aries-site/src/pages/templates/code-blocks.mdx

```diff
@@ -0,0 +1,56 @@
+import { Example } from '../../layouts';
+import {
+  CodeBlockExample
+} from '../../examples';
+
+<Example
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/code-blocks/CodeBlocks.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/code-blocks/DarkCodeTheme.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/code-blocks/LightCodeTheme.js",
+  ]}
+  showResponsiveControls={['laptop', 'mobile']}
+  template
+>
+  <CodeBlockExample />
+</Example>
+
+
+## Guidance
+
+Code blocks should be created using the [react-syntax-highlighter](https://github.com/react-syntax-highlighter/react-syntax-highlighter) tool.
+
+Prism or Highlighter.js can be used with react-syntax-highlighter with Highlighter.js being the default. Highlighter.js occasionally has trouble highlighting jsx so it is recommended to use Prism instead. 
+The documentation for using Prism with react-syntax-highlighter includes a [list of available languages](https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/master/AVAILABLE_LANGUAGES_PRISM.MD).
+
+react-syntax-highlighter can be installed with the following command:
+
+For yarn users: `yarn add react-syntax-highlighter`
+
+For npm users: `npm install react-syntax-highlighter --save`
```

Comment:
Should we modify our [code Markdown styling](https://github.com/grommet/hpe-design-system/blob/77d3a3e7d035db81b37a97a6c970132f9eac0dd3/aries-site/src/components/content/MarkdownComponents.js#L32) so that it is wrapped in `<SyntaxHighlighter />`?

Could be nice :)

Reply:
I'm good with that.